src/screens/ContactScreen/index.js

```diff
@@ -129,14 +139,15 @@ const ContactScreen = ({navigation}) => {
       username={item.username}
       followed={extendedState.followed}
       userid={item.user_id}
+      isAnon={sourceScreen === ANONYMOUS}
       onPress={() => handleSelected(item)}
     />
   );
   // }
   const handleSelected = (value) => {
-    const copyFollowed = [...followed];
-    const copyUsername = [...usernames];
-    const copyUsers = [...selectedUsers];
+    const copyFollowed = sourceScreen === ANONYMOUS ? [] : [...followed];
+    const copyUsername = sourceScreen === ANONYMOUS ? [] : [...usernames];
+    const copyUsers = sourceScreen === ANONYMOUS ? [] : [...selectedUsers];
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `isAnon` prop is added to the `ItemUser` component and set to `sourceScreen === ANONYMOUS`. The `handleSelected` function is updated to initialize `copyFollowed`, `copyUsername`, and `copyUsers` as empty arrays if `sourceScreen === ANONYMOUS`, otherwise they are initialized as copies of `followed`, `usernames`, and `selectedUsers` respectively. These changes ensure that anonymous chats do not include followed users, usernames, or selected users.
```diff
+ isAnon={sourceScreen === ANONYMOUS}
+ onPress={() => handleSelected(item)}
+ />
+ );
+ // }
+ const handleSelected = (value) => {
+ const copyFollowed = sourceScreen === ANONYMOUS ? [] : [...followed];
+ const copyUsername = sourceScreen === ANONYMOUS ? [] : [...usernames];
+ const copyUsers = sourceScreen === ANONYMOUS ? [] : [...selectedUsers];
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->